beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -102,6 +104,15 @@ const WDMoveControls: React.FC<WDMoveControlsProps> = function ({
     0,
   );
 
+  if (ordersSaved) {
+    const disbandingOrdersSaved = ordersMetaValues.filter(
+      (val) => val.saved && val.update && val.update.type === "Disband",
+    );
+    if (disbandingOrdersSaved?.length) {
+      dispatch(gameApiSliceActions.updateUnitsDisbanding());
+    }
+  }
+
```

Comment:
This doesn't belong in WDMoveControls. This component should be concerned only with logic related to the save/ready buttons, not related to a particular phase of the game. You can add this kind of code in one of the extraReducers that listens to the result of saveOrders and that that particular order was saved successfully.